x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_status_tags.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { EuiBadge, EuiText } from '@elastic/eui';
+import { UptimeSettingsContext } from '../../../contexts';
+
+const TextStyle = styled.div`
+  text-transform: uppercase;
```

Comment:
Do we want this? The screenshot in the issue has a location (I think it was Japan) like `Japan`.

On my screen my locations render like this:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/18429259/71191208-f66a1a00-2253-11ea-975a-80f2853a7aca.png">

I found it kind of unsettling when I saw them in caps, as I've specified them like `Fairbanks` and `Us-east-2`.

Reply:
i agree, i also got chance to discuss with andrew, so have remote the caps.